source/Ship.cpp

```diff
@@ -1822,7 +1824,10 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 					if(altV > 0. && altV < exitV)
 						exitV = altV;
 				}
-				if(velocity.Length() <= exitV)
+				// Because velocity.Length() is not signed
+				// If velocity is lower than rate just consider it done so
+				// there's no chance it will skip over.
+				if(velocity.Length() <= HYPER_A)
```

Comment:
What happens with this instead?
```suggestion
				if(velocity.Dot(angle.Unit()) <= exitV)
```

Reply:
This seems to work as well from my quick test.